dusty/systems/docker/compose.py

```diff
@@ -63,14 +63,9 @@ def _restart_container(client, container):
 
     logging.info('Restarting service containers from list: {}'.format(services))
     client = get_docker_client()
-    dusty_containers = _get_dusty_containers(client, services)
-    expected_number_of_containers = get_expected_number_of_running_containers() if len(services) == 0 else len(services)
-    if len(dusty_containers) != expected_number_of_containers:
-        log_to_client("Not going to restart containers. Expected number of containers {} does not match {}".format(expected_number_of_containers, len(dusty_containers)))
-        raise RuntimeError("Please use `docker ps -a` to view crashed containers")
-    else:
-        for container in dusty_containers:
-            _restart_container(client, container)
+    dusty_containers = _get_dusty_containers(client, services, include_exited=True)
+    for container in dusty_containers:
+        _restart_container(client, container)
```

Comment:
You're right.  Is that the only reason the check was there?  Because I can implement a checker to see if containers to link to are started